server/chat.ts

```diff
@@ -515,6 +515,7 @@ export class CommandContext extends MessageContext {
 			}).catch(err => {
 				if (err.name?.endsWith('ErrorMessage')) {
 					this.errorReply(err.message);
+					this.update();
```

Comment:
I feel like I would expect `CommandContext#errorReply` to update. I suppose it's usually used at the end of a command so it doesn't matter but it seems odd. Is updating the room too much a performance issue?

Reply:
when i tested locally, broadcasting `!lastfm`, `!track`, etc. would not display the error message until I updated the room. I really didn't bother to look into it because I mostly did it to stop complaints of errors 'lagging', but I'd rather look into it more in-depth in another commit.

Reply:
This is the correct fix.

@AnnikaCodes The individual `reply`/`add`/`send` commands don't update, because it's more efficient (and saves bandwidth) to send one update with everything at once, than to send things one at a time.

Calling `update` here, at the end of the chat command handler, is the correct place to do it.

Reply:
Alright, that makes sense.